eclass/node-guru.eclass

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EXPORT_FUNCTIONS src_prepare src_compile src_install
+
+SLOT="0"
```

Comment:
```suggestion
```
should this be set in ebuilds?

Reply:
I don't plan on making slot aware packages for now